ChangeLog

```diff
@@ -39,6 +39,11 @@
     modifications to Hashtable that might exposure potential memory corruption 
     with default copy constructor
 
+2015-01-08  Reed Cartwright <cartwright@asu.edu>
```

Comment:
I like to see a ChangeLog in chronological order of when things are merged to the main branch: so this would go to the top of the Changelog with today's date.